### PR TITLE
Feat/updates for v5

### LINKED ISSUE #183: Updating for v5
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

I'm looking to update this repo for v5. https://github.com/fastify/fastify/issues/5116

Changes Needed (Please let me know if anything might be missing)

- Updaate dependencies
- Update workflows to use v4

### PR DESCRIPTION
**Pull request by nrayburn-tech**
Fixes #183.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by nrayburn-tech** [comment]
- [ ] Update target to `next`
- [ ] Update CI workflow to v4 tag

File: package.json [hunk 1]
```diff
@@ -33,34 +33,34 @@
   "scripts": {
     "lint": "eslint .",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap -J -R specy --no-coverage test/*test.js",
+    "test:unit": "tap --disable-coverage test/*test.js",
```
Comment (nrayburn-tech) [inline package.json:36]: `specy` isn't available in tap 18, so I just removed it and went with the default reporter.

Comment (gurgunday) [reply package.json:36]: It's alright, but please move `disable-coverage` and files to the `.taprc` file

```
disable-coverage: true
files:
  - test/**/*.test.js
```

Comment (nrayburn-tech) [reply package.json]: Updated.

**Review by gurgunday — APPROVED** [review]
lgtm

**Comment by gurgunday** [comment]
Need to sync next before merging

**Comment by gurgunday** [comment]
I'll sync next before merging

**Review by gurgunday — APPROVED** [review]

**Pull request merged**